docs/excel/excel-add-ins-pivottables.md

```diff
@@ -160,16 +249,17 @@ The currently supported aggregation function types are `Sum`, `Count`, `Average`
 
 The following code samples changes the aggregation to be averages of the data.
```

Comment:
"The following code samples..." should be singular -> "The following code sample..."